src/platform/nxp/crypto/se05x/BUILD.gn

```diff
@@ -23,12 +23,8 @@ if (chip_crypto == "platform") {
 }
 
 source_set("public_headers") {
-  sources = [
```

Comment:
Why did we remove sources here? 

A source set without sources seems suspicious. Should we remove the entire sourceset? It seems we just wanted to grab src/crypto headers instead?

Reply:
Fixed the issue. Removed the source set and added the required dependency.